apps/prairielearn/src/ee/lib/lti13.ts

```diff
@@ -857,10 +891,20 @@ export async function updateLti13Scores(
         body: JSON.stringify(score),
       });
 
-      job.info(`\t${res.statusText}`);
-      if (!res.ok) {
-        job.warn(`\t${await res.text()}`);
+      if (res.ok) {
+        counts.success++;
+      } else {
+        counts.error++;
+        const results = await res.json();
+
+        // Default to showing the whole error
+        job.warn(`\t${res.statusText}`);
+        job.warn(`\t${JSON.stringify(results)}`);
       }
     }
   }
+  job.info('Done.\n\nSummary:');
+  job.info(`${counts.success} score${counts.success === 1 ? '' : 's'} successfully sent.`);
+  job.info(`${counts.error} error${counts.error === 1 ? '' : 's'} sent.`);
```

Comment:
Nit, I don't like "1 error sent" - that implies that an error itself was somehow sent. Maybe "1 score not sent due to errors." instead?

Reply:
Oooh, agree. Changed in 41e2f91762